Week-3/Exercise-3.1/memoize.js

```diff
@@ -0,0 +1,34 @@
+function memoize(fn) {
+    const cache = new Map();
+    return function (...args) {
+      const key = args.toString();
+      console.log("Fn called with " + key);
+      console.log(cache);
+      if (args.length != 2) {

```

Comment:
Does this condition make any difference? What if we remove this condition will anything change?

Reply:
> Does this condition make any difference? What if we remove this condition will anything change?

Actually this condition do not make any difference in case of output. we can remove this function.

Reply:
Changes made by removing function in https://github.com/pesto-students/p9-manish-anandpaher/pull/4/files/f761085d2910d3bd8456c784ce02b234760ff22f#diff-f6dcd42b302080c6058ed5ff45dc7a713e2ca4486d03c19ff71a4d344c635ea2